stoobly_agent/test/config/data_dir_test.py

```diff
@@ -0,0 +1,78 @@
+import os
+import shutil
+
+from stoobly_agent.config.data_dir import DataDir
+from stoobly_agent.test.test_helper import reset
+
+
+class TestDataDir():
+  original_cwd = os.getcwd()
+
+  def test_in_home(self):
+    # A previous test can put us in a test folder
+    os.chdir(self.original_cwd)
+    reset()
```

Comment:
Would make use a of a fixture for this to act like a `before each` e.g. https://github.com/Stoobly/stoobly-agent/blob/746471b1306e18d5d84441f1bff57633b5cd06fc/stoobly_agent/test/app/cli/request/delete_test.py#L16

You may want to use `class` scope though